frontend/src/app/edit-product/edit-product.component.html

```diff
@@ -0,0 +1 @@
+<p>edit-product works!</p>
```

Comment:
is the EditProduct component used? This looks like the initialized file when it's created from ng serve

Reply:
The EditProduct Component isn't being used, I was going to use it to display the edit product area but then realized I didn't know where/how I would pass through the product I was editing and then just decided to go with editing the product form the admin area.

Reply:
Ok, then can you delete this component and remove it from the app routing module? You can push that change and it’ll update this PR